ProjectSidewalk/SidewalkWebpage
# 72 resume mission at same location

Resolves #72 

When you return to the audit page, you are now placed in the location you were when you last left it. It lets you resume an audit task (street). This will decrease the amount of data we lose, since users don't start over on a new street when they return to the website. This also fixes the problem of really long streets never being finished.

Things to test on the audit page:
1. Try going less than halfway down the street and refreshing the page. Make sure you are placed back in the same location, with the green part of the route on the mini map being on the same side of you.
1. Test the same as above, but after going more than halfway down the street (I had to be careful when recording which endpoint we started at, which is why this needs to be tested).
1. Place a few labels, refresh the page, place a few more labels. Make sure that all the labels are recorded, are associated with the same `audit_task_id` in the database, and have distinct `temporary_label_id`s.
1. Test previous items when there is a mission that is completed in the middle of the audit task to make sure that doesn't mess things up.
1. Go around and audit for awhile, making sure nothing is broken :)
